src/main/scala/org/broadinstitute/dsde/firecloud/service/WorkspaceService.scala

```diff
@@ -63,21 +55,17 @@ class WorkspaceService(protected val argUserToken: WithAccessToken,
 
   implicit val userToken: WithAccessToken = argUserToken
 
+  val priceList = Map("STANDARD" -> 0.02, "NEARLINE" -> 0.01, "COLDLINE" -> 0.004, "ARCHIVE" -> 0.0012, "REGIONAL" -> 0.02, "MULTI-REGIONAL" -> 0.02) //TODO: dma?
```

Comment:
Whoops, TODO ["dra"](https://cloud.google.com/storage/docs/storage-classes) not "dma".  Not sure what the price for that would be, or what exactly it would be coded as when returned from the google metrics API.  Seems like I should have a catch-all for any values that are not in this list, but unclear what the cost should be.  Ideas?